src/applications/mhv/medical-records/containers/Vaccines.jsx

```diff
@@ -107,6 +107,40 @@ const Vaccines = props => {
     makePdf(pdfName, pdfData, 'Vaccines', runningUnitTest);
   };
 
+  const generateVaccinesTxt = async () => {
+    const product = `
+    Vaccines\n 
+    For a list of your allergies and reactions (including any reactions to
+    vaccines), go to your allergy records. \n
+    If you have Vaccines that are missing from this list, tell your care
+    team at your next appointment. \n
+    
+    Showing ${vaccines.length} from newest to oldest. \n
+    ${vaccines.map(
+      entry => `_____________________________________________________ \n
+      ${entry.name} \n
+      \t Date entered: ${entry.date} \n
+      \t Signs and symptoms: ${entry.reaction} \n
+      \t Type of Vaccines: ${entry.type} \n
+      \t Location: ${entry.location} \n
+      \t Observed or historical: ${entry.observedOrReported} \n
+      \t Provider notes: ${entry.notes} \n`,
```

Comment:
These fields should be the same fields that appear in the vaccine PDF. Same for Detail view.